src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -5,6 +5,9 @@ var f = require('{0}'),
     util = require('util');
 
 return function (context, callback) {{
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
+    setImmediate(() => {{}});
```

Comment:
I'll admit that if this truly works in all scenarios, then it feels safer than replacing nextTick as we were doing previously. However, might this again be a breaking change for people? I.e. scenarios that are currently running (fixed with the previous fix) now start failing?